runtime/src/server/middleware/get_page_handler.ts

```diff
@@ -230,24 +231,28 @@ export function get_page_handler(
 				error.stack = sourcemap_stacktrace(error.stack);
 			}
 
+			const pageContext: PageContext = {
+				host: req.headers.host,
+				path: req.path,
+				query: req.query,
+				params,
+				error: error ? error instanceof Error ? error : { message: error } : null,
```

Comment:
I think to keep the types simpler and more consistent I would change this to `{ name: 'PreloadError', message: error }` and then change the type of `PageContext.error` to be just `Error` instead of `Error | { message: string; }`

Reply:
Ok, sure! Please also check my comments and suggestions above on this.